website/docs/blueprints/inference/GPUs/vLLM-rayserve.md

```diff
@@ -76,10 +76,7 @@ For example, set your `export AWS_DEFAULT_REGION="<REGION>"` to the desired regi
 **Step2**: Run the installation script.
 
 ```bash
-cd ai-on-eks/infra/jark-stack/terraform && chmod +x install.sh
-```
-
-```bash
+cd ai-on-eks/infra/jark-stack/ && chmod +x install.sh
```

Comment:
Could you please split this PR and raise this change a separate PR? 

I don't really see a need for adding g6e Karpenter nodepool without showing any examples using this new nodepool. May be we can show this [example](https://community.aws/content/2w2T9a1HOICvNCVKVRyVXUxuKff/deploying-the-deepseek-v3-model-full-version-in-amazon-eks-using-vllm-and-lws) written by **Davanum Srinivas** as a second PR. I can check with Dims to see if he is ok with that.

Reply:
works for me!